learn.qmd

```diff
@@ -1,13 +1,73 @@
 ---
-title: "Learn"
-listing:
-  contents: learn
-  sort: "date desc"
-  type: default
-  categories: true
-  sort-ui: false
-  filter-ui: false
-  feed: true
-page-layout: full
-title-block-banner: true
+format:
+  html:
+    toc: true
 ---
+
+# Learning materials
+
+We organise our training materials in two curricula. The **Outbreak Analytics curriculum** aims to facilitate finding, choosing, and using R packages for outbreak analytics tasks to inform decision-making. The Research **Sofware curriculum** aims to extend our R toolset to produce open, reproducible and sustainable data analysis projects following good Research Sofware practices.
+
+We address these goals with an e-Epi training kit, Tutorials,How-to guides, and Case studies:
```

Comment:
For consistency with the order of listed elements:

```suggestion
We address these goals with an e-Epi training kit, Tutorials, Case studies, and How-to guides:
```

Reply:
Included 👍